index.html

```diff
@@ -602,8 +603,9 @@ <h1>Trust Model</h1>
 risk assessment of the verifier.
         </li>
         <li>
-All entities trust the <a>identifier registry</a> to be incorruptible and
-to be a correct record of which identifiers belong to which <a>entities</a>.
+All entities trust the <a>Public data repository</a> to be un-corruptible and
```

Comment:
typo `un-corruptible` => `incorruptible` ... or maybe just use `tamper-evident` instead like we do elsewhere?

Reply:
Done